frontend/src/features/permits/components/permit-list/BasePermitList.tsx

```diff
@@ -144,14 +144,14 @@ export const BasePermitList = ({
         return (
           <Box sx={{ display: "flex", justifyContent: "flex-end" }}>
             {DoesUserHaveAuthGroup({
-              userAuthGroup: idirUserDetails?.userAuthGroup,
+              userAuthGroup: idirUserDetails?.userRole,
               allowedAuthGroups: [IDIR_USER_AUTH_GROUP.PPC_CLERK],
```

Comment:
Consider renaming "DoesUserHaveAuthGroup", "userAuthGroup", and "allowedAuthGroups" to "DoesUserHaveRole", "userRole", and "allowedRoles" respectively

Reply:
and the enum "IDIR_USER_AUTH_GROUP" to "IDIR_USER_ROLES"